packages/vite/src/node/optimizer/index.ts

```diff
@@ -1271,8 +1271,11 @@ export async function cleanupDepsCacheStaleDirs(
       for (const dirent of dirents) {
         if (dirent.isDirectory() && dirent.name.includes('_temp_')) {
           const tempDirPath = path.resolve(config.cacheDir, dirent.name)
-          const { mtime } = await fsp.stat(tempDirPath)
-          if (Date.now() - mtime.getTime() > MAX_TEMP_DIR_AGE_MS) {
+          const stats = await fsp.stat(tempDirPath).catch((_) => null)
```

Comment:
`tempDirPath` maybe removed when `server.close()`, so add a `catch` to make clean up process work properly